src/components/actions/scheduler.ts

```diff
@@ -2,6 +2,7 @@ import DBConnect from '../databaseConnect'
 import scheduler from 'node-schedule'
```

Comment:
Just FYI -- I plan to move this file out of the "actions" folder into a more general location, since it contains schedulers for more than just actions now.

However, I thought I'd wait until after Fergus's "file clean-up" script is done, since that adds another scheduler here, and the conflicts would be a headache if I've already moved this file.